tweet_on_upcoming_and_recently_released_movies_using_azure_openai/main.bal

```diff
@@ -0,0 +1,67 @@
+import ballerina/io;
+import ballerinax/azure.openai.text;
+import ballerinax/themoviedb;
+import ballerinax/twitter;
+
+configurable string moviedbApiKey = ?;
+
+configurable string openAIToken = ?;
+configurable string serviceUrl = ?;
+configurable string deploymentId = ?;
+
+configurable string twitterApiKey = ?;
+configurable string twitterApiSecret = ?;
+configurable string twitterAccessToken = ?;
+configurable string twitterAccessTokenSecret = ?;
+
+const API_VERSION = "2023-03-15-preview";
+const MAX_TWEET_LENGTH = 280;
+const MAX_TOKENS = 1000;
+const NO_OF_MOVIES = 5;
+
+// Twitter API client configuration
+twitter:ConnectionConfig twitterConfig = {
+    apiKey: twitterApiKey,
+    apiSecret: twitterApiSecret,
+    accessToken: twitterAccessToken,
+    accessTokenSecret: twitterAccessTokenSecret
+};
```

Comment:
Can replace with below line and remove line 12:15 and line 23:27

```ballerina
configurable twitter:ConnectionConfig twitterConfig = ?;
```

Reply:
Addressed in https://github.com/ballerina-guides/ai-samples/pull/63/commits/51d5361f5867dd852237e579e3c1e9938ebaf8ae.